### PR TITLE
opentx: update implementation according to rfc (command: 1)

### PR DESCRIPTION
**Pull request by joii2020**

### CODE REVIEW

File: tests/omni_lock_rust/tests/test_opentx.rs [hunk 1]
```diff
@@ -626,3 +671,62 @@ fn test_opentx_item_missing() {
     let verify_result = run_opentx_case(config);
     check_res_val(verify_result, vec![1]);
 }
```
Comment (XuJiandong) [inline tests/omni_lock_rust/tests/test_opentx.rs:673]: Add test cases when any of len(input, output, group input, group output) is zero.

**Review by XuJiandong — CHANGES_REQUESTED** [review]

**Review by XuJiandong — APPROVED** [review]
